src/content/history/index.md

```diff
@@ -58,7 +58,7 @@ Be sure to upgrade your client software to the latest version before December 5,
 
 #### Summary {#altair-summary}
 
-The Altair upgrade was the first scheduled upgrade for the [Beacon Chain](/eth2/beacon-chain). It added support for "sync committees"—enabling light clients, and bringing validator inactivity and slashing penalties up to their full values.
+The Altair upgrade was the first scheduled upgrade for the [Beacon Chain](/eth2/beacon-chain). It added support for "sync committees" – enabling light clients, and bringing validator inactivity and slashing penalties up to their full values.
```

Comment:
This is the correct usage of a long dash (no spaces on either side)

More info on [long dash usage](https://www.grammarly.com/blog/why-you-should-love-the-em-dash/#:~:text=Use%20an%20Em%20Dash%20to%20Bring%20Focus%20to%20a%20List,the%20list%20to%20the%20clause.)

```suggestion
The Altair upgrade was the first scheduled upgrade for the [Beacon Chain](/eth2/beacon-chain). It added support for "sync committees"—enabling light clients, and bringing validator inactivity and slashing penalties up to their full values.
```